### PR TITLE
Add ACES endpoints

### PR DESCRIPTION
**Pull request by epelaaez**
(related: https://github.com/Infleqtion/server-superstaq/pull/2612)

### CODE REVIEW

File: qiskit-superstaq/qiskit_superstaq/superstaq_provider.py [hunk 1]
```diff
@@ -469,6 +469,59 @@ def process_dfe(self, ids: List[str]) -> float:
         """
         return self._client.process_dfe(ids)
 
+    def submit_aces(
```
Comment (richrines1) [inline qiskit-superstaq/qiskit_superstaq/superstaq_provider.py:472]: it looks like this is the same in css.Service and qss.SuperstaqProvider - should we just move it to `gss.service.Service` instead?

also we should add it to `qss.SuperstaqBackend` (which would just be a wrapper for `self._provider(target=self.name(), ...)`)

Comment (epelaaez) [reply qiskit-superstaq/qiskit_superstaq/superstaq_provider.py:472]: > it looks like this is the same in css.Service and qss.SuperstaqProvider - should we just move it to gss.service.Service instead?

sounds good

> also we should add it to qss.SuperstaqBackend (which would just be a wrapper for self._provider(target=self.name(), ...))

will do

Comment (epelaaez) [reply qiskit-superstaq/qiskit_superstaq/superstaq_provider.py]: done!

**Review by richrines1 — APPROVED** [review]
lgtm!

**Comment by epelaaez** [comment]
> lgtm!

cool! will merge once https://github.com/Infleqtion/server-superstaq/pull/2612 goes in

**Pull request merged**